src/components/spinner/index.tsx

```diff
@@ -2,22 +2,26 @@ import { Component, h, Host, Prop } from "@stencil/core"
 
 @Component({
 	tag: "smoothly-spinner",
-	styleUrl: "style.scss",
+	styleUrl: "style.css",
 	scoped: true,
 })
 export class SmoothlySpinner {
-	@Prop({ reflect: true }) active: boolean
-	@Prop({ reflect: true }) size: "small" | "medium" | "large" = "large"
+	@Prop({ reflect: true }) size: "small" | "icon" | "medium" | "large" = "large"
+	@Prop({ reflect: true }) overlay: boolean
 
 	render() {
-		const strokeWidth = this.size == "large" ? 6 : this.size == "medium" ? 8 : 12
+		const strokeWidth = this.size == "large" ? 6 : this.size == "medium" ? 8 : this.size == "icon" ? 8 : 12
 		return (
 			<Host
 				style={{
-					"--color": `var(--spinner-color)`,
-					"--size": this.size == "large" ? "5em" : this.size == "medium" ? "3em" : "1.2em",
+					"--spinner-size":
+						this.size == "large" ? "5em" : this.size == "medium" ? "3em" : this.size == "icon" ? "1.8em" : "1.2em",
 				}}>
-				<svg class="spinner" viewBox={`0 0 ${60 + strokeWidth} ${60 + strokeWidth}`} xmlns="http://www.w3.org/2000/svg">
+				<svg
+					part="spinner"
```

Comment:
Does this attribute matter? it only works for shadow components iirc?

Reply:
Yep. Removed.